sys/arvo.hoon

```diff
@@ -651,8 +800,100 @@
   ::
   [[[~ %vega hap] ((list ovum) -.out)] +.out]
 ::
-++  veer                                                ::  install vane/tang
-  |=  {now/@da fav/curd}
+++  velo                                                ::  new full reboot
+  |=  $:  ::  now: current date
+          ::  ova: actions to process after reboot
+          ::  hun: hoon.hoon source
+          ::  arv: arvo.hoon source
+          ::
+          now=@da
+          ova=(list ovum)
+          hun=@t
+          van=@t
+      ==
+  ^-  (unit (pair (list ovum) *))
+  ::  virtualize; dump error if we fail
```

Comment:
This is a useful pattern, and I think it deserves its own arm in hoon.hoon. Out of scope for this PR.

Reply:
Cosign